packages/offsite-integration/jest.config.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
     globals: {
         'ts-jest': {
             tsconfig: '<rootDir>/tsconfig.spec.json',
+            diagnostics: false,
```

Comment:
What does this change?

Reply:
This change removes jest errors what occurs due to global LIBRARY_VERSION variable in payment-integration-api package